pkg/v22/key/key.go

```diff
@@ -83,6 +83,14 @@ func ClusterOrganization(clusterGuestConfig v1alpha1.ClusterGuestConfig) string
 // CommonAppSpecs returns apps installed for all providers.
 func CommonAppSpecs() []AppSpec {
 	return []AppSpec{
+		{
+			App:             "cert-exporter",
+			Catalog:         "default",
+			Chart:           "cert-exporter",
+			Namespace:       metav1.NamespaceSystem,
+			UseUpgradeForce: true,
+			Version:         "1.2.0",
```

Comment:
@sslavic After https://github.com/giantswarm/cluster-operator/pull/752 this can be merged. 

But the version bundles have been reset to empty so this may need adjusting.

Reply:
Rebased, PTAL